java-lib/src/main/java/com/yammer/metrics/core/WavefrontHistogram.java

```diff
@@ -115,8 +116,8 @@ private MinuteBin getCurrent() {
     // bins with clear == true flag will drain (CONSUMER) the list,
     // so synchronize the access to the respective 'bins' list
     synchronized (bins) {
-      if (bins.isEmpty() || bins.getLast().minMillis != currMinMillis) {
-        bins.add(new MinuteBin(currMinMillis));
+      if (bins.isEmpty() || bins.getLast().getMinuteMillis() != currMinMillis) {
```

Comment:
Let's rename this to "currMinuteMillis" and elsewhere.
i.e. change minMillis to minuteMillis across the board.